string awake_word
---
string result
string fail_reason 
